typescript/sdk/src/middleware/account/types.ts

```diff
@@ -0,0 +1,9 @@
+import { Address } from '@hyperlane-xyz/utils';
+
+import { ChainName } from '../../types';
+
+export type AccountConfig = {
+  origin: ChainName;
+  owner: Address;
+  localRouter?: Address;
```

Comment:
this is mixing config and artifacts

Reply:
yaa but this was the easiest route for now

Reply:
you should have the ICA artifacts as deploy option, following the pattern of the ISM factories